frontend/public/locales/fr/learn.json

```diff
@@ -3,70 +3,70 @@
       "cards": [
         {
-          "title": "(FR) Main sources of retirement income",
-          "read": "(FR) 5 MIN READ",
-          "body": "(FR) Learn about public & private pensions, personal savings and investments.",
+          "title": "Principales sources de revenus de retraite",
+          "read": "Temps de lecture : 9 minutes",
+          "body": "Renseignez-vous sur les prestations de vieillesse, les régimes privés d'épargne-retraite et la manière dont ils fonctionnent ensemble",
           "image": "/assets/main-sources-of-retirement-income-sm.jpg",
           "link": "/learn/main-sources-of-retirement-income"
         },
         {
-          "title": "(FR) Planning to save for retirement",
-          "read": "(FR) 5 MIN READ",
-          "body": "(FR) Learn how to maximize your savings and to better manage your money in retirement.",
+          "title": "Planifiez pour épargner en vue de la retraite",
+          "read": "Temps de lecture: 5 minutes",
+          "body": "Découvrez comment maximiser votre épargne et mieux gérer votre argent à la retraite.",
           "image": "/assets/planning-to-save-for-retirement-sm.jpg",
           "link": "/learn/planning-to-save-for-retirement"
         }
       ]
     },
     {
-      "title": "(FR) Making retirement decisions",
-      "body": "(FR) Gain clarity and confidence when making retirement decisions with our comprehensive section on CPP and OAS pension timing and transitioning from work, designed to support and empower you on your retirement journey.",
+      "title": "Prendre des décisions concernant la retraite",
+      "body": "Soyez plus confiant dans les choix que vous faites concernant votre retraite. Utilisez ces ressources pour mieux comprendre les options qui s'offrent à vous et leur incidence sur vos prestations.",
       "cards": [
         {
-          "title": "(FR) When to take government pensions",
-          "read": "(FR) 5 MIN READ",
-          "body": "(FR) Lorem ipsum dolor sit amet consectetur. Gravida eget tristique in ipsum eget ornare libero.",
+          "title": "Quand commencer à recevoir des pensions gouvernementales",
+          "read": "Temps de lecture : 9 minutes",
+          "body": "Vous pouvez choisir le moment où vous souhaitez commencer à recevoir vos prestations de vieillesse. Assurez-vous de bien comprendre les éléments dont il faut tenir compte pour prendre cette décision.",
           "image": "/assets/deciding-when-to-collect-public-pensions-sm.jpg",
           "link": "/learn/deciding-when-to-collect-public-pensions"
         },
         {
-          "title": "(FR) Going from work to retirement",
-          "read": "(FR) 7 MIN READ",
-          "body": "(FR) Lorem ipsum dolor sit amet consectetur. Gravida eget tristique in ipsum eget ornare libero.",
+          "title": "Working during retirement",
```

Comment:
Missed translation here, or missing `(FR)` to indicate that it has yet to be translated.

Reply:
Pre-fix with (FR) awaiting translation.